src/modules/calls/calls.controller.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  Controller,
+  Get,
+  Body,
+  HttpStatus,
+  UseInterceptors,
+  Req,
+  Res,
+  Param,
+  Put,
+  ClassSerializerInterceptor,
+  Post,
+} from '@nestjs/common';
+import {
+  ApiResponse,
+  ApiParam,
+  ApiUnauthorizedResponse,
+  ApiTags,
+} from '@nestjs/swagger';
+
+//import  VoiceResponse  from 'twilio/twiml/VoiceResponse';
+const VoiceResponse = require('twilio').twiml.VoiceResponse;
+import { CallsService } from './calls.service';
+import { CallQueryDto } from './dto/call-query.dto';
+import { ConfigurationService } from 'src/configuration/configuration.service';
+
+
+@Controller('call')
+@ApiTags('Calls')
+@ApiUnauthorizedResponse({ description: 'Unauthorized' })
+export class CallsController {
+  constructor(private readonly callService: CallsService,
+    private readonly configService: ConfigurationService) { }
+
+  @Post('twilio/call')
```

Comment:
ich vermute in dem endpunkt sollte noch nen secret im header gecheckt werden, obs auch wirklich von twilio kommt.

Reply:
Ja fügen wir noch hinzu